tests/qgis-expected.yaml

```diff
@@ -69,6 +69,8 @@ metadata:
     app.kubernetes.io/name: apache
     app.kubernetes.io/instance: qgis
     app.kubernetes.io/component: main
+  annotations:
+    reloader.stakater.com/auto: "true"
```

Comment:
for info: what is this?

Reply:
As example, I use the annotation used by https://github.com/stakater/Reloader#how-to-use-reloader